projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -52,12 +61,23 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   private _panelIndex: number;
 
   constructor(
+    @Optional()
+    @SkipSelf()
+    protected parent: ClrAccordionPanel,
```

Comment:
```suggestion
    private parent: ClrAccordionPanel,
```

This can be `private`. 1) The override in the stepper is not needed since this is an accordion-only feature. 2) Even if we did support this feature in the stepper, you do not need to override the property. You just need to pass a parameter to the `super` constructor.

Reply:
Also, I would put this on one line like everything else.

```
@Optional() @SkipSelf() private parent: ClrAccordionPanel,
```

Reply:
Corrwcted in commit 2cb2400